src/ask_order.rs

```diff
@@ -73,126 +53,10 @@ pub fn migrate_ask_orders(
-}
-
 #[cfg(test)]
-mod tests {
-    use cosmwasm_std::{coin, Addr, Uint128};
-    use cosmwasm_storage::{bucket, Bucket};
-    use provwasm_mocks::mock_dependencies;
-
-    #[allow(deprecated)]
-    use crate::ask_order::{
-        get_ask_storage_read, migrate_ask_orders, AskOrder, AskOrderClass, AskOrderV1,
-        NAMESPACE_ORDER_ASK,
-    };
-    use crate::contract_info;
-    use crate::contract_info::set_legacy_contract_info;
-    use crate::error::ContractError;
-    use crate::msg::MigrateMsg;
-
-    #[test]
-    #[allow(deprecated)]
-    pub fn migrate_legacy_ask_to_v1() -> Result<(), ContractError> {
-        let mut deps = mock_dependencies(&[]);
-
-        set_legacy_contract_info(
-            &mut deps.storage,
-            &contract_info::ContractInfo {
-                name: "contract_name".to_string(),
-                definition: "contract_def".to_string(),
-                version: "0.14.99".to_string(),
-                bind_name: "bind_name".to_string(),
-                base_denom: "base_1".to_string(),
-                convertible_base_denoms: vec![],
-                supported_quote_denoms: vec![],
-                executors: vec![],
-                issuers: vec![],
-                ask_required_attributes: vec![],
-                bid_required_attributes: vec![],
-                price_precision: Uint128::new(2),
-                size_increment: Uint128::new(100),
-            },
-        )?;
-
-        let mut legacy_ask_storage: Bucket<AskOrder> =
-            bucket(&mut deps.storage, NAMESPACE_ORDER_ASK);
-
-        legacy_ask_storage.save(
-            b"id",
-            &AskOrder {
-                base: coin(100, "base_1"),
-                class: AskOrderClass::Basic,
-                id: "id".to_string(),
-                owner: Addr::unchecked("asker"),
-                price: "10".to_string(),
-                quote: "quote_1".to_string(),
-                size: Uint128::new(100),
-            },
-        )?;
-
-        migrate_ask_orders(
-            deps.as_mut(),
-            &MigrateMsg {
-                approvers: None,
-                ask_fee_rate: None,
-                ask_fee_account: None,
-                bid_fee_rate: None,
-                ask_required_attributes: None,
-                bid_required_attributes: None,
-                bid_fee_account: None,
-            },
-        )?;
-
-        let ask_storage = get_ask_storage_read(&deps.storage);
-        let migrated_ask = ask_storage.load(b"id")?;
-
-        assert_eq!(
-            migrated_ask,
-            AskOrderV1 {
-                id: "id".to_string(),
-                owner: Addr::unchecked("asker"),
-                class: AskOrderClass::Basic,
-                base: "base_1".to_string(),
-                quote: "quote_1".to_string(),
-                price: "10".to_string(),
-                size: Uint128::new(100)
-            }
-        );
-
-        Ok(())
-    }
-}
+mod tests {}
```

Comment:
Technically, i think you could make a test where you call setup an old version_info  and `migrate_ask_orders()` and see that it fails

Reply:
yeah, you're right. I did that for bid orders